draft-ietf-httpbis-http2bis.xml

```diff
@@ -4333,30 +4187,6 @@
             </dd>
         </dl>
       </section>
-      <section anchor="iana-h2c">
-        <name>The h2c Upgrade Token</name>
```

Comment:
As this registry doesn't have a status column, we can leave this untouched, with the reference to 7540 left alone.